user_guide/business_intelligence.rst

```diff
@@ -0,0 +1,23 @@
+.. _business-intelligence:
+
+Business Intelligence Tools
+===========================
+
+OroCRM provides a set of functionalities for data analysis and decision-making. With numerous dedicated widgets the users can 
```

Comment:
suggestion: OroCRM comes with a set of tools for data analysis and decision-making.
suggestion: OroCRM comes with a set of data analysis tools to help users make more informed decisions.